framework/generated/khronos_generators/vulkan_generators/vulkan_struct_handle_wrappers_body_generator.py

```diff
@@ -233,37 +262,6 @@ def need_feature_generation(self):
             return True
         return False
 
-    def generate_feature(self):
```

Comment:
Same comment.  Rename the function, touch up the variable names in place, and then call from endfile.  Easier to review, more maintainable.